app/models/case_log.rb

```diff
@@ -395,10 +395,10 @@ def is_prevten_la_general_needs?
 
   def self.to_csv
     CSV.generate(headers: true) do |csv|
-      csv << attribute_names
+      csv << attribute_names + %w[unittype_sh]
 
       all.find_each do |record|
-        csv << record.attributes.map do |att, val|
+        csv << record.attributes.merge({ "unittype_sh" => record.unittype_sh }).map do |att, val|
```

Comment:
Given "We’ll also want all the other fields from the scheme and from the location" it might be easier to just loop over scheme and location fields entirely as well.

Reply:
Will look at adding a test, probably worth looping, but perhaps when we know exactly how the export is gonna look like?